clmm/theory/cluster_toolkit.py

```diff
@@ -81,8 +81,6 @@ def eval_3d_density(self, r3d, z_cl):
                 self.cdelta, Omega_m, delta=self.delta_mdef)*h**2
 
     def eval_surface_density(self, r_proj, z_cl):
-        if self.cosmo is None:
-            raise ValueError(f"Missing cosmology.")
         h = self.cosmo['h']
```

Comment:
Is that check not needed anymore? Or did you move it somewhere else?

Reply:
This check is not really needed here, I am guessing it was a leftover from some previous version